### PR TITLE
i18n(es): Update `routing`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### Description (required)

Update `routing` based on this commit c5c074b15f9f89d0750462f22f77045aac6045c5

#### Related issues & labels (optional)

- Suggested label: i18n

### CODE REVIEW

**Review by Waxer59 — APPROVED** [review]
LGTM ✅

File: src/content/docs/es/core-concepts/routing.mdx [hunk 1]
```diff
@@ -290,32 +290,34 @@ if (!isLoggedIn(cookie)) {
...
-Astro necesita saber qué ruta debe usarse para construir la página. Para ello, los ordena de acuerdo con las siguientes reglas:
+Astro necesita saber qué ruta debe usarse para construir la página. Para ello, los ordena de acuerdo con las siguientes reglas en orden:
 
-- Las rutas estáticas sin parámetros de ruta tendrán prioridad sobre todas las demás rutas
-- Las rutas dinámicas que usan parámetros nombrados tienen prioridad sobre los parámetros rest
-- Las rutas dinámicas pre-renderizadas tienen prioridad sobre las rutas dinámicas del servidor
-- Los parámetros rest tienen la prioridad más baja
-- Los endpoints siempre tienen prioridad sobre las páginas
-- Los empates se resuelven alfabéticamente
+- Las rutas con más segmentos de ruta tendrán prioridad sobre las rutas con menos parámetros. En el ejemplo anterior, `/posts/[page].astro` tiene prioridad sobre `/posts/[pid].astro`.
```
Comment (Waxer59) [inline src/content/docs/es/core-concepts/routing.mdx:307]:
```suggestion
- Las rutas con más segmentos de ruta tendrán prioridad sobre las rutas con menos parámetros. En el ejemplo anterior, todas las rutas bajo `/posts/` tienen prioridad sobre `/[...slug].astro` en la raíz.
```

**Review by Waxer59 — COMMENTED** [review]
Revisando el commit de actualización me he fijado en esta parte

**Comment by dreyfus92** [comment]
Muchas gracias por el review @Waxer59 😁

**Pull request merged**